test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -57,6 +57,49 @@
         x "x-"]
     (str r w x)))
 
+(def attribute-file-extensions
+  "These are the extensions defined in csr_attributes.yaml."
```

Comment:
s / csr_attributes.yaml / (something like) "the test csr_attributes.yaml used by this namespace"